Taskfile.yml

```diff
@@ -382,7 +386,9 @@ tasks:
   ci:reset:
     desc: Create CI setup in a clean state
     cmds:
-      - task dev:reset
+      - task dev:reset:common
+      - task dev:cli -- drush sqlq --file=../tools/site-reset.sql.gz
```

Comment:
What is contained in site-reset.sql.gz? 
It would be nice if it was documented somewhere what is happening here